### ISSUE TITLE
Accessibility Audit: Results page user flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5775]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the Results page user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/results`

Use cases to test:
* view results page with no, 1, and multiple results
* view results page with multiple pages
* result details modal
* download test results modal
* print/email/text result modals

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by johanna-skylight** [comment]
Tickets created to track the issues found:

https://github.com/CDCgov/prime-simplereport/issues/5860
https://github.com/CDCgov/prime-simplereport/issues/5861
https://github.com/CDCgov/prime-simplereport/issues/5862
